site/content/integrate/admin-guide/admin-bot-accounts/_index.md

```diff
@@ -142,7 +142,10 @@ Replace the following parameters:
 
 ### Do bot access tokens expire?
 
-No, but you can automate your integration to cycle its token [through the REST API](https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D~1tokens%2Fpost).
+No, but you can automate your integration to cycle its token through the REST API:
+1. Use [CreateUserAccessToken](https://api.mattermost.com/#operation/CreateUserAccessToken) to generate a new token.
+2. Update the services that leverage the token with the new `token` value from step 1
+3. Use RevokeUserAccessToken](https://api.mattermost.com/#operation/RevokeUserAccessToken) to revoke the old token based on the `token_id`.
```

Comment:
We can add another bullet here to inform about listing tokens from https://api.mattermost.com/#operation/GetUserAccessTokensForUser so that they can find the token IDs.

Reply:
Additional bullet added.  Thanks @isacikgoz